projects/1.20/assets/missing-wilds-forge/missingwilds/lang/zh_cn.json

```
@@ -0,0 +1,228 @@
+{
+  "itemGroup.missingwilds.items": "遗落荒野",
+  "block.missingwilds.fallen_acacia_log": "倒下的金合欢原木",
+  "block.missingwilds.fallen_birch_log": "倒下的白桦原木",
+  "block.missingwilds.fallen_jungle_log": "倒下的丛林原木",
+  "block.missingwilds.fallen_oak_log": "倒下的橡木原木",
+  "block.missingwilds.fallen_dark_oak_log": "倒下的深色橡木原木",
+  "block.missingwilds.fallen_spruce_log": "倒下的云杉原木",
+  "block.missingwilds.fallen_crimson_stem": "倒下的绯红菌柄",
+  "block.missingwilds.fallen_warped_stem": "倒下的诡异菌柄",
+  "block.missingwilds.fallen_mushroom_stem": "倒下的蘑菇柄",
+  "block.missingwilds.fallen_mangrove_log": "倒下的红树原木",
+  "block.missingwilds.fallen_cherry_log": "倒下的樱花原木",
+  "block.missingwilds.blue_forget_me_not": "蓝色勿忘我",
+  "block.missingwilds.purple_forget_me_not": "紫色勿忘我",
+  "block.missingwilds.pink_forget_me_not": "粉红色勿忘我",
+  "block.missingwilds.white_forget_me_not": "白色勿忘我",
+  "block.missingwilds.brown_polypore_mushroom": "棕色多孔菌",
+  "item.missingwilds.roasted_polypore_mushroom": "烤多孔菌",
+  "block.missingwilds.sweetspire": "北美鼠刺",
+  "block.missingwilds.potted_sweetspire": "北美鼠刺盆栽",
+  "block.missingwilds.potted_blue_forget_me_not": "蓝色勿忘我盆栽",
+  "block.missingwilds.potted_pink_forget_me_not": "粉红色勿忘我盆栽",
+  "block.missingwilds.potted_purple_forget_me_not": "紫色勿忘我盆栽",
+  "block.missingwilds.potted_white_forget_me_not": "白色勿忘我盆栽",
+  "stat.missingwilds.log_crawl_one_cm": "在原木中爬行距离",

```

```suggestion
  "stat.missingwilds.log_crawl_one_cm": "原木中爬行距离",

```
向旧版本同步，统计数据的话加个“在”还是有些画蛇添足